cmd/ocm-backplane/console/console.go

```diff
@@ -637,14 +659,22 @@ func (o *consoleOptions) cleanUp(ce containerEngineInterface) error {
 		o.terminationFunction = &execActionOnTermStruct{}
 	}
 
+	<-ctx.Done()
 	err = o.terminationFunction.execActionOnTerminationFunction(func() error {
 		for _, c := range containersToCleanUp {
-			err := ce.stopContainer(c)
+			exist, err := ce.containerIsExist(c)
 			if err != nil {
-				return fmt.Errorf("failed to stop container %s: %v", c, err)
+				return fmt.Errorf("failed to check container exist %s: %v", c, err)
+			}
+			if exist {
+				err := ce.stopContainer(c)
+				if err != nil {
+					return fmt.Errorf("failed to stop container %s: %v", c, err)
```

Comment:
how about add some more context for the return errors?
```
failed to stop container '%s' during cleanup process. Ensure the container engine is running and accessible
```

Reply:
Update error message, As I noted, `%v`  will throw the detailed exception as to why it failed